framework/src/Volo.Abp.Json/Volo/Abp/Json/SystemTextJson/JsonConverters/AbpStringToEnumConverter.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Volo.Abp.Json.SystemTextJson.JsonConverters
+{
+    public class AbpStringToEnumConverter : JsonConverter<object>
+    {
+        private readonly JsonStringEnumConverter _innerJsonStringEnumConverter;
+
+        public AbpStringToEnumConverter()
+            : this(namingPolicy: null, allowIntegerValues: true)
+        {
+        }
+
+        public AbpStringToEnumConverter(JsonNamingPolicy namingPolicy = null, bool allowIntegerValues = true)
+        {
+            _innerJsonStringEnumConverter = new JsonStringEnumConverter(namingPolicy, allowIntegerValues);
+        }
+
+        public override bool CanConvert(Type typeToConvert)
+        {
+            return typeToConvert.IsEnum;
+        }
+
+        public override object Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            var newOptions = new JsonSerializerOptions(options);
+            newOptions.Converters.Remove(this);
```

Comment:
I didn't understand what we are trying to do. We first add this converter to JsonSerializerOptions.Converters, then removing here. Why this workaround-like code here?

Reply:
If we don't remove `this(AbpStringToEnumConverter)`, it will cause infinite loop. We hope it uses `innerJsonStringEnumConverter`.